modules/src/url.py

```diff
@@ -33,8 +34,13 @@ def process(input, entities):
             data = r.json()
             response = 'Here\'s your original URL:\n' + url
             response += '\nHere\'s your shortened URL:\n' + data['id']
+
         output['input'] = input
-        output['output'] = TextTemplate(response).get_message()
+        template = TextTemplate(response)
+        text = template.get_text()
+        template = ButtonTemplate(text)
+        template.add_web_url('Click to visit URL', url)
```

Comment:
In the shorten query, the button needs to point to shortened URL while in expand query, it needs to point to the expanded URL.